### DISCUSSION TITLE
qwen2.5的prompt和llamafactory不一致

### DISCUSSION THREAD

**Discussion by HiXiaochen** [QwenLM/Qwen3#1140] [General]
看官方demo，qwen2.5的prompt为："You are Qwen, created by Alibaba Cloud. You are a helpful assistant." 然后推荐用llamafactory进行sft，并指定template参数为qwen，但看llamafactory的代码，qwen模板的指令为"You are a helpful assistant." 如果直接用llamafacotry的这个模板，请问是不是会对效果有点影响？是否建议修改为官方示例中的prompt？

**Comment by jklj077** [comment]
影响有限吧，保持训练推理一致其实就好。

**Reply by HiXiaochen** [reply]
我对比下看看 感觉instruction tuning的时候用的prompt和sft不一致的话可能会有一点影响
